build.gradle

```diff
@@ -27,9 +27,9 @@ intellij {
 }
 
 dependencies {
-    compile 'io.fabric8:openshift-client:5.0.0'
-    compile 'io.fabric8:tekton-client:5.0.0'
-    compile 'com.redhat.devtools.intellij:intellij-common:0.0.1-SNAPSHOT'
+    compile 'io.fabric8:openshift-client:5.0.2'
+    compile 'io.fabric8:tekton-client:5.0.2'
+    compile 'com.redhat.devtools.intellij:intellij-common:0.0.1'
```

Comment:
Shouldn't it be 0.0.2-SNAPSHOT? Otherwise the IJ-common uses kubernetes-client:5.0.0 and IJ-tekton *-client:5.0.2?

Reply:
My plan is to use 0.0.1 then release 0.0.2 using 5.0.2.
I also checked that even using 0.0.1/5.0.0, F8 libraries used by IJ Tekton are 5.0.2 as it's a direct dependency